resources/js/post/post.js

```diff
@@ -5,5 +5,27 @@
  * the SSO
  */
 ready(function () {
-    document.getElementById('postLoginSubmitButton').click();
+    // Reveal #slowPostMessage after a delay (default 30000 ms).
+    // If you want to change the delay per page, set data-slow-post-delay on the element.
+    const el = document.getElementById('slowPostMessage');
+    const pageTitle = document.getElementById('page-title');
+    if (!el) return;
```

Comment:
Does this mean if someone has customized post.twig and doesn't have these new elements, then the javascript to autoclick no longer runs?  I think we would want the autosubmit to always be called.